packages/common/src/content/_internal.ts

```diff
@@ -181,6 +181,20 @@ export const getHubRelativeUrl = (
 export const isPageType = (type: string, typeKeywords: string[] = []) =>
   ["Hub Page", "Site Page"].includes(type) || typeKeywords.includes("hubPage");
 
+/**
+ * Determines whether, given a type and typekeywords, the input is
+ * a site item type or not
+ * @param type - the type value on the item
+ * @param typeKeywords - the typeKeywords on the item
+ */
+
+export function isSiteType(type: string, typeKeywords: string[] = []) {
+  return (
+    type === "Site Application" ||
```

Comment:
There is one other critical branch we = need to include, which is `type === "Hub Site Application"`. It's not explicit in `normalizeItemType`, but you'll notice that `Hub Site Application` is what the types are being normalized to.